src/test/java/com/werfen/report/GridReportTest.java

```diff
@@ -94,6 +94,16 @@ public void generateGridXlsxReport() {
         } catch (JRException | IOException e) {
             e.printStackTrace();
         }
+
+        try (Workbook original = new XSSFWorkbook(new File("grid_report_golden.xlsx"));
+             Workbook generated = new XSSFWorkbook(new File("grid_report.xlsx"))) {
```

Comment:
Generated grid report filepath string shouldn't be duplicated here. For maintenance purposes, I would suggest taking it from the configuration object above.